src/Microsoft.AspNet.Mvc.Razor/Resources.resx

```diff
@@ -148,7 +148,7 @@
     <value>There is no active writing scope to end.</value>
   </data>
   <data name="RazorPage_YouCannotFlushWhileInAWritingScope" xml:space="preserve">
-    <value>You cannot flush while inside a writing scope.</value>
+    <value>You cannot flush while inside a writing scope. (Path: '{0}')</value>
```

Comment:
also suggest `'{0}' invocation in '{1}' is invalid. That method must not be used inside a writing scope.`